### PR TITLE
fix(cli): Drop 3.7, See #8698

### PR DESCRIPTION
**Pull request by ajbt200128**
No more python 3.7!

### CODE REVIEW

**Comment by ajbt200128** [comment]
See #8698

File: .github/workflows/build-test-manylinux-x86.yml [hunk 1]
```diff
@@ -12,7 +12,7 @@ jobs:
       with:
         submodules: true
     - name: Setup Python
-      run: "\n        rm /usr/bin/python\n        ln `which python3.7` /usr/bin/python3\n
+      run: "\n        rm /usr/bin/python\n        ln `which python3.8` /usr/bin/python3\n
```
Comment (aryx) [inline .github/workflows/build-test-manylinux-x86.yml]: please modify the .jsonnet and regenerate the .yml (not the other way around).

**Review by aryx — APPROVED** [review]

File: .github/workflows/build-test-manylinux-x86.jsonnet [hunk 2]
```diff
@@ -13,19 +13,20 @@ local wheel_name = 'manylinux-x86-wheel';
 local build_wheels_job = {
   'runs-on': 'ubuntu-latest',
   // pad: What is this sgrep-xxx image?
-  container: 'returntocorp/sgrep-build:ubuntu-16.04',
+  container: 'returntocorp/sgrep-build:ubuntu-18.04',
```
Comment (aryx) [inline .github/workflows/build-test-manylinux-x86.jsonnet:16]: Can we use a more official container? I have no idea what is this sgrep-build thing.
(sgrep was the old name of semgrep, but I don't even remember building those containers).

**Review by aryx — COMMENTED** [review]
nice!

File: .github/workflows/build-test-manylinux-x86.jsonnet (hunk 2 above)
Comment (ajbt200128) [reply .github/workflows/build-test-manylinux-x86.jsonnet:16]: Yea probably, I think this is the equivalent of the current ocaml-layer image?

**Pull request merged**